_layouts/getstarted.html

```diff
@@ -9,6 +9,23 @@ <h1 class="text-caps">{{page.title}} {{page.docversion}}</h1>
     <div class="grid__item width-12-12 width-12-12-mobile">
         <div>
             {{ content }}
+            {% comment %}
+            {% This should be part of the content in geststarted.adoc %}
+            {% But since we want to use the variable for the latest WildFly version %}
+            {% to render the link to the OS guide, this is part of the layout %}
+            {% endcomment %}
+            {% assign latestRelease = site.data.releases[site.current_release_index] %}
```

Comment:
For this we'll need something more sophisticated, as the 'latest' release often has no associated docs content; e.g. betas and micros have no docs content specific to those versions.

Reply:
FWIW I do something similar on resteasy.dev. https://github.com/resteasy/resteasy.github.io/blob/9ea62676d6ab77974ae4b6dfacf2d78f4d7530de/docs.html#L11-L13 is how I do the look up and the data is layout https://github.com/resteasy/resteasy.github.io/blob/main/_data/releases.yaml#L1.